advocacy_docs/partner_docs/ImpervaDataSecurityFabric/05-UsingImpervaDataSecurityFabric.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Using Imperva Agent with EDB Postgres Advanced Server'
+description: 'Walkthroughs of multiple Imperva usage scenarios'
```

Comment:
Change to "Walkthroughs of multiple Imperva Data Security Fabric usage scenarios"